### PR TITLE
Fixed broken links in article

### PR DESCRIPTION
**Pull request by ganesh-sanap**
## Category

- [x] Content fix

## What's in this Pull Request?

> **DELETE THIS LINE BEFORE SUBMITTING** | *Please describe the changes in this PR. Sample description or details around bugs which are being fixed.*

## Submission guidelines

Fixed broken link to other article in "See also" section.

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[94e6706](https://github.com/ganesh-sanap/sp-dev-docs/commits/94e6706cfb5c1d30be7f92a2d8838033287cb014)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/solution-guidance/Handle-SharePoint-Online-throttling-by-using-exponential-back-off.md](https://github.com/ganesh-sanap/sp-dev-docs/blob/patch-7/docs/solution-guidance/Handle-SharePoint-Online-throttling-by-using-exponential-back-off.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/solution-guidance/handle-sharepoint-online-throttling-by-using-exponential-back-off?branch=pr-en-us-6583) | [Details](#user-content-89c4f01f-53257f1c)

<a id="89c4f01f-53257f1c"></a>
### [docs/solution-guidance/Handle-SharePoint-Online-throttling-by-using-exponential-back-off.md](https://github.com/ganesh-sanap/sp-dev-docs/blob/patch-7/docs/solution-guidance/Handle-SharePoint-Online-throttling-by-using-exponential-back-off.md)
 - **Line 2, Column 1**: **[Suggestion-description-missing]** `````Missing required attribute: 'description'.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C12%5C28%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202012280907567491-6583%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Z3ADEXTCkQetXVt0AMfw2VNB2kaVLpEbgtr%2BNyEJrow%3D&st=2020-12-28T09%3A04%3A05Z&se=2021-01-28T09%3A09%3A05Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by andrewconnell** [comment]
Same comment... While the link is fixed, this should remove the link, not point to a now-removed sample. Can you please update this to remove the link to the sample as it's no longer there?

File: docs/solution-guidance/Handle-SharePoint-Online-throttling-by-using-exponential-back-off.md [hunk 1]
```diff
@@ -13,18 +13,18 @@ _**Applies to:** Office 365 | SharePoint Online | SharePoint Server 2013_
...
 
 SharePoint Online uses throttling to prevent users from over-consuming resources. When a user runs CSOM or REST code that exceeds usage limits, SharePoint Online throttles any further request from the user for a period of time.
 
-The [Core.Throttling](https://github.com/SharePoint/PnP/tree/master/Samples/Core.Throttling) code sample in the [SharePoint Patterns and Practices](https://github.com/SharePoint/PnP) organization shows how to implement the exponential back off technique to handle throttling in SharePoint Online. When you get throttled in SharePoint Online, the exponential back off technique waits progressively longer periods of time before retrying the code that was throttled.
+The [Core.Throttling](https://github.com/pnp/PnP/tree/master/Samples/Core.Throttling) code sample in the [SharePoint Patterns and Practices](https://github.com/pnp/PnP) organization shows how to implement the exponential back off technique to handle throttling in SharePoint Online. When you get throttled in SharePoint Online, the exponential back off technique waits progressively longer periods of time before retrying the code that was throttled.
```
Comment (andrewconnell) [inline docs/solution-guidance/Handle-SharePoint-Online-throttling-by-using-exponential-back-off.md:20]: Remove link... sample was removed.

**Review by andrewconnell — CHANGES_REQUESTED** [review]

File: docs/solution-guidance/Handle-SharePoint-Online-throttling-by-using-exponential-back-off.md (hunk 1 above)
Comment (ganesh-sanap) [reply docs/solution-guidance/Handle-SharePoint-Online-throttling-by-using-exponential-back-off.md:20]: @andrewconnell Should I remove complete paragraph or just first sentence in paragraph?

Comment (andrewconnell) [reply docs/solution-guidance/Handle-SharePoint-Online-throttling-by-using-exponential-back-off.md:20]: I'd rewrite the paragraph to remove the two links, but instead, reference the primary doc for this topic: https://docs.microsoft.com/en-us/sharepoint/dev/general-development/how-to-avoid-getting-throttled-or-blocked-in-sharepoint-online

Comment (ganesh-sanap) [reply docs/solution-guidance/Handle-SharePoint-Online-throttling-by-using-exponential-back-off.md:20]: Sure, thank you @andrewconnell.

**Review by andrewconnell — APPROVED** [review]

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[9fd5a15](https://github.com/ganesh-sanap/sp-dev-docs/commits/9fd5a15619b83e0e6de328eba7c158d6425d9c38)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/solution-guidance/Handle-SharePoint-Online-throttling-by-using-exponential-back-off.md](https://github.com/ganesh-sanap/sp-dev-docs/blob/patch-7/docs/solution-guidance/Handle-SharePoint-Online-throttling-by-using-exponential-back-off.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/solution-guidance/handle-sharepoint-online-throttling-by-using-exponential-back-off?branch=pr-en-us-6583) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2021%5C1%5C18%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202101181105511427-6583%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=xyq%2BANB8IA3HR4Ff6lpA%2BHBJrnzHFZQLuSgWLeWMZxk%3D&st=2021-01-18T11%3A01%3A51Z&se=2021-02-18T11%3A06%3A51Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**